Documentation/project-docs/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,119 @@
+We expect the first version of the .NET Core tools that use MSBuild to ship along with Visual Studio "15".  However, it is possible to use MSBuild for .NET Core
+projects today, and this page shows how.
+
+We recommend that most people targeting .NET Core today use the default tooling experience with project.json.  This is because we haven't yet added support to MSBuild
+for a lot of the benefits that project.json has, because a lot of the ASP.NET based tooling will not work with MSBuild today, and because when we do release .NET Core
+tooling which uses MSBuild, it will be able to automatically convert from project.json to MSBuild based projects.
+
+You may want to consider using MSBuild to target .NET Core for existing projects that already use MSBuild that you want to port to .NET Core, or if you are using
+MSBuild's extensibility in your build for scenarios that are not well supported for project.json projects.
+
+Prerequisites
+=============
+
+- [Visual Studio 2015 Update 2 or higher](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- [.NET Core tools for Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- NuGet Visual Studio extension [v3.5.0-beta](https://dist.nuget.org/visualstudio-2015-vsix/v3.5.0-beta/NuGet.Tools.vsix) or later
+
+Creating a library targeting .NET Core
+======================================
+
+- File > New > Project
+
+  ![New Project](https://dotnetdocs.blob.core.windows.net/getting-started/new-project.png)
+
+- Select ".NET Framework 4.6" and "ASP.NET Core 1.0"
+
+  ![Portable targets dialog](pcl-targets-dialog-net46-aspnetcore10.png)
+
+- In the "Library" tab of the project properties, click on the "Target .NET Platform Standard" link, and click "Yes" in the dialog that is shown
+- In the `project.json` file:
+    - Change the version number of the `NETStandard.Library` package to `1.5.0-rc2-24027` (this is the .NET Core RC2 version of the package).
+    - Add the below `imports` definition inside the `netstandard1.5` framework definition.  This will allow your project to reference .NET Core compatible
+      NuGet packages that haven't been updated to target .NET Standard
+
+        ```json
+        "netstandard1.5": {
+            "imports": [ "dnxcore50", "portable-net452" ]
+        }
+        ```
+
+- You may see an error in the Visual Studio error list saying "Platform target 'AnyCPU' is not supported by one or more of the project's targets."  If so,
+you can suppress the error by editing the `.csproj` file and removing the line with the `ProjectTypeGuids` property  
+
+Creating a .NET Core console application
+========================================
+Building a console application for .NET Core requires some customization of the MSBuild build process.  A good option to start with is
+[coretemplate](https://github.com/mellinoe/coretemplate).  This is a basic project which brings in MSBuild targets for .NET Core.  
```

Comment:
I have updated the sample at https://github.com/dotnet/corefxlab/tree/master/samples/NetCoreSample, based on RC2 so it all works as well. I've not looked into @mellinoe sample deeply but the sample in corefxlab doesn't require any extra targets just some hacking in the project files themselves and give examples for out to cross-compile file x-plat.